mmo.cc

```diff
@@ -259,9 +260,9 @@ mmo_register_object(const int objid, const void* ptr, const gpsdata_type type)
 static int
 mmo_get_objid(const void* ptr)
 {
-  foreach(int key, objects.keys()) {
-    if (objects.value(key)->data == ptr) {
-      return key;
+  for (auto o = objects.constBegin(); o != objects.constEnd(); ++o) {
```

Comment:
This code has probably just been mechanically converted a few too many times. Is this:
if objects[ptr].has_value() return object[ptr]
This looks like some of Olaf's home grown map code.

Reply:
Your translation doesn't look good to me.

Codacy shows we do have coverage on those lines.

If this web site is related to this format, then I wonder about the usefulness of us supporting it.
http://memory-map.com/mm-PC-help2/index.htm#t=Maptech_io.htm
shows they support csv, gpx, shape files(with professional license), and loc files besides their proprietary format.